x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/creation_mode_components/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { ShowCreationModes } from './show_creation_modes';
+export { ApiCodeExample } from './api_code_example';
+export { PasteJsonText } from './paste_json_text';
+export { UploadJsonFile } from './upload_json_file';
```

Comment:
FYI: I plan to have other response views (e.g., success/error summaries) in a separate sub-component folder from this (I'm thinking `creation_response_components`?), just because it can be a bit of a headache to find things once you start getting 7-8+ components in the same folder